### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

**Comment by Zapadlo** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -40,4 +45,13 @@ will result in
...
 ```
 
+Username and password should be set at `/ft/_credentials/vulcand/` like it was mentioned above, to be able to set for the marked services.
 These routing rules will change as we develop. The idea is they are in a single place in this application, not spread out across many unmaintainable sidekick services.
+
+## Test the app locally
+
+1. Install [__etcd__](https://github.com/coreos/etcd)
+2. `go get github.com/Financial-Times/vulcan-config-builder`
+3. `git clone https://github.com/Financial-Times/vulcan-config-builder.git && cd vulcan-config-builder`
```
Comment (Nesze) [inline README.md]: Just for clarity: you don't actually need both 'go get' and 'git clone'. Running 'go build' is also redundant here.

File: main.go [hunk 2]
```diff
@@ -117,6 +120,11 @@ func readServices(kapi client.KeysAPI) []Service {
 				for _, path := range child.Nodes {
 					service.PathPrefixes[filepath.Base(path.Key)] = path.Value
 				}
+			case "auth":
+				service.NeedsAuthentication, err = strconv.ParseBool(child.Value)
+				if (err != nil) {
```
Comment (Nesze) [inline main.go]: No need for paranthesis.

**Comment by Nesze** [comment]
+1

File: README.md (hunk 1 above)
Comment (kovacshuni) [inline README.md]: Removed some.

File: main.go (hunk 2 above)
Comment (kovacshuni) [inline main.go]: Removed paranthesis.

**Pull request merged**